src/MainModules/ProcessingModules/PlainCodeGenerator.java

```diff
@@ -0,0 +1,29 @@
+package MainModules.ProcessingModules;
+
+import lombok.SneakyThrows;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileWriter;
+
+public class PlainCodeGenerator {
+    @SneakyThrows
+    public void start()
+    {
+        File file = new File("./codes/generated.java");
+        if (!file.exists()) file.createNewFile();
+        FileWriter fileWriter = new FileWriter(file);
+
+        String generatedText =
```

Comment:
"""
                        public class Main{
                            public static void main(String[] args){
                                System.out.println("Hello world!");
                            }
                        }""";
                        
change this